.goreleaser.yml

```diff
@@ -0,0 +1,52 @@
+project_name: driftctl
+dist: bin
+release:
+    github:
+        owner: cloudskiff
+        name: driftctl
+    draft: false
+    prerelease: auto
+before:
+    hooks:
+        - go mod tidy
+        - go generate ./...
+signs:
+    - signature: "${artifact}.gpg"
+      artifacts: checksum
+      args: [ "--batch", "-u", "{{ .Env.SIGNINGKEY }}", "--output", "${signature}", "--detach-sign", "${artifact}" ]
+builds:
+    - id: "driftctl"
+      binary: driftctl
+      dir: .
+      env:
+          - CGO_ENABLED=0
+      goos:
+          - linux
+          - windows
+          - darwin
+      goarch:
+          - amd64
+          - arm
+          - arm64
+          - 386
+      ldflags: "-s -w -X github.com/cloudskiff/driftctl/build.env={{ .Env.ENV }} -X github.com/cloudskiff/driftctl/pkg/version.version={{ .Tag }}"
+archives:
+    - name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}'
+      format: binary
+      replacements:
+          armv6: arm
+checksum:
+    name_template: '{{ .ProjectName }}_SHA256SUMS'
+    algorithm: sha256
+snapshot:
+    name_template: "{{ .Tag }}-next"
+nfpms:
+    - maintainer: CloudSkiff <engineering@cloudskiff.com>
+      description: Detect, track and alert on infrastructure drift
+      homepage: https://github.com/cloudskiff/driftctl
+      license: Apache License 2.0
+      file_name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}'
+      formats:
```

Comment:
I don't think that we need deb, apk and rpm packages right now since we do not have a release process on theses platform. It's good to know that we can easily generate them but we'll do the work in dedicated PR in the future.

Reply:
Ok should I comment this for now ?